cdap-ui/app/features/apps/templates/list.html

```diff
@@ -4,9 +4,28 @@
     <h2> All Applications </h2>
   </div>
   <div class="col-xs-6 h2 text-right">
-    <my-file-select on-file-select="onFileSelected($files)"
-      data-button-label="Add App"
-      data-button-icon="fa-plus"> </my-file-select>
+    <div class="btn-group" dropdown="">
```

Comment:
Do we need ```dropdown=""``` Can't we just have 

```<div class="btn-group" dropdown>
      <button type="button" dropdown-toggle>
      ...
   ...
```
We can specify them as tags since its an angular directive (I suppose we are using angular bootstrap's dropdown)